packages/contracts/contracts/tutorials/Arbitrage.sol

```diff
@@ -0,0 +1,103 @@
+pragma solidity ^0.4.19;
+pragma experimental ABIEncoderV2;
+
+import "../Exchange.sol";
+import "../tokens/Token.sol";
+import "./EtherDelta.sol";
+import "../tokens/WETH9.sol";
+
+contract Arbitrage is Ownable {
+
+    Exchange exchange;
+    EtherDelta etherDelta;
+    address proxyAddress;
+
+    uint constant MAX_UINT = 2**256 - 1;
```

Comment:
I know our other contracts use `uint` instead of `uint256`, but I prefer the latter going forward to make things more explicit (applies to all of contract).